src/groupeddataframe/grouping.jl

```diff
@@ -61,10 +61,10 @@ and combines the result into a data frame).
 
 See the following for additional split-apply-combine operations:
 
-* `by` : split-apply-combine using functions
-* `aggregate` : split-apply-combine; applies functions in the form of a cross product
-* `map` : apply a function to each group of a `GroupedDataFrame` (without combining)
-* `combine` : combine a `GroupedDataFrame`, optionally applying a function to each group
+* [`by`](@ref) : split-apply-combine using functions
+* [`aggregate`](@ref) : split-apply-combine; applies functions in the form of a cross product
+* [`map`](@ref) : apply a function to each group of a `GroupedDataFrame` (without combining)
```

Comment:
Just please confirm that after building the documentation correct links are generated (this is tricky sometimes). In particular does `map` point to a proper docstring? Thank you!

Reply:
Will do...let me check this locally / consult the Documenter docs.  Might take me a bit of time, a little preoccupied at the moment :)

Reply:
No rush - thank you. I have just made a release today, so probably it is at least another 2 weeks till the next release.

Reply:
I've just tried it, and it doesn't seem to have any effect (surprisingly). @mortenpi Could you help us?

BTW, you could remove the `[Returns](#returns)` above, which creates an invalid link.

Reply:
I built this branch locally and, as far as I can tell, the at-ref links work fine.

Reply:
Thank you - so you mean that `map` is a link to `map` method defined for `GroupedDataFrame` (not a general `map`). This would mean that Documenter.jl looks for a docstring within a package by default - right?

Reply:
Ah, sorry for bothering you, I must have used the wrong branch. I can see links now, and they point to DataFrames docstrings, so all is well!

Reply:
@bkamins Documenter does not know how to link across packages, so yes. E.g. the `map` link will point to here: http://juliadata.github.io/DataFrames.jl/dev/lib/functions/#Base.map